Sources/Scheduler.swift

```diff
@@ -85,7 +90,11 @@ public final class UIScheduler: SchedulerProtocol {
 			                               value: dispatchSpecificValue)
 	}()
 
+	#if os(Linux)
+	private var queueLength: Atomic<Int32> = Atomic(0)
```

Comment:
I'm not sure if this is a right way. Please advise me if there is a better way to implement the functionality. 🙏